tests/ixia/README.md

```diff
@@ -0,0 +1,9 @@
+# Important Note for Contributors
+
+Test cases in this directory have been migrated to /tests/snappi. **Since December 2022, Snappi has become our primary API (through `snappi`) for interacting with Keysight Ixia devices**, while we are deprecating the existing `ixia` API from our test repository. Any new test cases from contributors requiring the use of any *Ixia* devices should utilise the `snappi` API, and any new pull requests which still calls the `ixia` API will be disregarded.
```

Comment:
s/keysight Ixia devices/hardware traffic generators/ and s/Ixia devices/hardware traffic generators/